auditlog/models/rule.py

```diff
@@ -735,3 +735,10 @@ def _update_vals_list(self, vals_list):
                 if isinstance(fieldvalue, models.BaseModel) and not fieldvalue:
                     vals[fieldname] = False
         return vals_list
+
+    def _update_registry(self):
+        """Force a registry reload after rule change"""
+        # this code comes from `base_automation` which has a similar need
+        if self.env.registry.ready and not self.env.context.get("import_file"):
+            # notify other workers
+            self.env.registry.registry_invalidated = True
```

Comment:
Is it something compatible on Odoo < 17.0 that should be backported?

Reply:
That seems at least compatible with Odoo 16.0, didn't checked previous versions.